Cargo.toml

```diff
@@ -56,13 +56,11 @@ chrono = "0.4.39"
 regex = "1.11.1"
 
 [dev-dependencies]
-async-trait = "0.1.88"
```

Comment:
`#[tonic::async_trait]` only modifies the structure of a statement, which makes imports unnecessary.